apps/settings/src/router.js

```diff
@@ -53,10 +53,26 @@ export default new Router({
 			component: Users,
 			props: true,
 			name: 'users',
+			meta: {
+				title: () => {
+					return t('settings', 'Active users')
+				},
+			},
 			children: [
 				{
 					path: ':selectedGroup',
 					name: 'group',
+					meta: {
+						title: (to) => {
+							if (to.params.selectedGroup === 'admin' ) {
```

Comment:
```suggestion
							if (to.params.selectedGroup === 'admin') {
```
to make eslint happy